### DISCUSSION TITLE
How to select content inside "" , '' , {}, (), [], etc

### DISCUSSION THREAD

**Discussion by andradei** [helix-editor/helix#6748] [Q&A] [answered]
Say I have the string `"this is some text"` in a file I'm editing.

In vim, if I want to replace it with new text that I'll type, I'd do `ci"` and start typing.

How can I do something similar in Helix?

**Comment by andradei** [comment]
Found it. It's `mi"`

m - Match
i - Inside object

**Comment by Diorgeles** [comment, marked as answer]
> Say I have the string `"this is some text"` in a file I'm editing.
> 
> In vim, if I want to replace it with new text that I'll type, I'd do `ci"` and start typing.
> 
> How can I do something similar in Helix?
Hello, 
In helix, `mic` or `ma[` etc
read this https://docs.helix-editor.com/keymap.html#match-mode
